document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/SmallFullAccountsDataHandler.java

```diff
@@ -107,6 +123,23 @@ private AccountType getCompanyAccountType(CompanyAccounts accountsData) throws H
                         accountsData.getKind()));
     }
 
+    private CompanyAccounts getCompanyAccounts(String resourceUri, String requestId) throws HandlerException {
+
+        CompanyAccounts accounts;
```

Comment:
This could be simplified by moving the variables into the scope they are used:
```java
try {
            return accountsService.getCompanyAccounts(resourceUri, requestId);
} catch (ServiceException e) {
            Map<String, Object> logMap = new HashMap<>();
            logMap.put(RESOURCE, resourceUri);
            LOG.errorContext(requestId, "Error obtaining company-accounts data", e, logMap);
            throw new HandlerException(e.getMessage(), e.getCause());
}
```

Reply:
not for doc-gen though? as it is now in a new class called CompanyAccountsDataHandler